src/schema/convert-edtr-io-state.tsx

```diff
@@ -72,8 +200,8 @@ function convertPlugin(node: EdtrState) {
     return [
       {
         type: 'img',
-        src: node.state.src,
-        alt: node.state.alt,
+        src: node.state.src as string,
+        alt: node.state.alt || '',
```

Comment:
Here was a mismatch between types and actual implementation. Feel free to revert that change and instead make `alt`. optional